conmon-rs/server/src/child_reaper.rs

```diff
@@ -46,7 +47,7 @@ macro_rules! lock {
 
 impl ChildReaper {
     pub fn get(&self, id: &str) -> Result<ReapableChild> {
-        let locked_grandchildren = Arc::clone(&self.grandchildren);
+        let locked_grandchildren = &self.grandchildren().clone();
```

Comment:
there was a reason we went for Arc::clone, @cgwalters I believe it was yours

Reply:
both syntaxes are correct. [docs](https://doc.rust-lang.org/std/sync/struct.Arc.html#deref-behavior).

I do prefer the .clone() one. It looks more idiomatic to me.